modules/cart/src/Form/AddToCartForm.php

```diff
@@ -85,6 +85,34 @@ public static function create(ContainerInterface $container) {
   /**
    * {@inheritdoc}
    */
+  public function getBaseFormId() {
+    $form_id = $this->entity->getEntityTypeId();
+    if ($this->entity->getEntityType()->hasKey('bundle')) {
+      $form_id .= '_' . $this->entity->bundle();
+    }
+    return $form_id . '_form';
```

Comment:
Ensure's base form ID is always `commerce_line_item_BUNDLE_form` but maybe we should add `operation` instead of bundle?